subprojects/parseq-tracevis/package.json

```diff
@@ -9,7 +9,7 @@
     "jshint": "~2.1.11",
     "istanbul": "~0.1.44",
     "mocha": "1.2.x",
-    "phantomjs": "~1.9.2-2",
+    "phantomjs": "^2.1.7",
```

Comment:
For my own understanding, what does ^  mean here? 2.1.7 or above?

Reply:
equivalent to 2.*
~2.1.7  is 2.1.*

Reply:
I see. Thanks.